packages/cli/src/commands/dev.ts

```diff
@@ -30,7 +34,59 @@ export async function devCommand(options: DevCommandOptions) {
 
   const httpServer = new DevServer(opts);
 
+  const studioSpinner = ora('Starting local studio server').start();
   await httpServer.listen();
+  studioSpinner.succeed(`Novu Studio started: ${httpServer.getStudioAddress()}`);
+
+  const NOVU_ENDPOINT_PATH = '/api/novu';
+
+  if (process.env.NODE_ENV !== 'dev') {
+    await open(httpServer.getStudioAddress());
+  }
+
+  let healthy = false;
+  const endpointText = `Looking for the Novu Endpoint at ${parsedOptions.origin}${NOVU_ENDPOINT_PATH}. Ensure your application is configured and running locally.`;
+  const endpointSpinner = ora(endpointText).start();
+
+  let counter = 0;
+  while (!healthy) {
+    try {
+      const response = await fetch(`${parsedOptions.origin}${NOVU_ENDPOINT_PATH}?action=health-check`, {
```

Comment:
Nit pick: we should prefer fetch over axios, so we can remove axios in a follow up PR.

Reply:
Im pretty sure i used fetch here 😀

Reply:
Correct, but axios is already in CLI. This comment was for a follow up PR.